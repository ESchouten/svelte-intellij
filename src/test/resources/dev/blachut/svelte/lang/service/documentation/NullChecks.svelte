<script lang="ts">
  /**
   * docs have to be removed!
   */
  let local = 323 as number | null;
</script>

{<caret>local}