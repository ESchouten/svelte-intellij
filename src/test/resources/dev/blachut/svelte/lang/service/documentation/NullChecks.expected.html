let local: number | null<br/>[ide]<br/>let local: number